L1Trigger/L1TGlobal/src/CorrCondition.cc

```diff
@@ -1149,7 +1166,7 @@ const bool l1t::CorrCondition::evaluateCondition(const int bxEval) const {
         }
       }
 
-      if (corrPar.corrCutType & 0x8 || corrPar.corrCutType & 0x10) {
+      if (corrPar.corrCutType & 0x8 || corrPar.corrCutType & 0x10 || corrPar.corrCutType & 0x40) { // R. Cavanaugh added 0x40 for massUpt
```

Comment:
Please remove meaningless comments. Git versioning system keeps track of authors of commits.

Reply:
@rekovic 
Many thanks for the helpful request to remove comments.  Done.